recipes/libxlsxwriter/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class LibxlsxwriterConan(ConanFile):
+    name = "libxlsxwriter"
+    license = "FreeBSD"
```

Comment:
```suggestion
    license = "BSD-2-Clause"
```

Reply:
Should it be `BSD-2-Clause-FreeBSD` ?

https://github.com/jmcnamara/libxlsxwriter/blob/ff35a56934f7bb528f8f3d35debbd4569628a50d/License.txt#L5

https://spdx.org/licenses/BSD-2-Clause-FreeBSD.html